src/Transformation/TransformationTransformerTrait.php

```diff
@@ -13,27 +13,28 @@ public function transform($data, Context $context = NULL) {
     if (!isset($context)) {
       $context = new Context();
     }
-    if (!$this->conformsToExpectedInputShape($data, $context)) {
+
+    // Error utility.
+    $throwError = function($message, $arguments = []) {
```

Comment:
Let's use snake case for all variable names. We will be observing Drupal Coding Standards in this library.